# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: config/system-headers.mozbuild
# Commit: 5147903580b8
# Full Hash: 5147903580b8f39197d67bddd5a0fb91d3670da4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-27 04:32:10
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r 0aad3393b572 -r 5147903580b8 config/system-headers.mozbuild
--- a/config/system-headers.mozbuild	Mon Feb 26 16:07:07 2024 +0000
+++ b/config/system-headers.mozbuild	Mon Feb 26 16:07:08 2024 +0000
@@ -1267,11 +1267,6 @@
         "jpeglib.h",
     ]
 
-if CONFIG["MOZ_LIBAV_FFT"]:
-    system_headers += [
-        "libavcodec/avfft.h",
-    ]
-
 if CONFIG["MOZ_SYSTEM_PNG"]:
     system_headers += [
         "png.h",